### ISSUE TITLE
Compatibility with OneWire library?

### ISSUE THREAD

**Issue by miqi95** [yoelk/instrumentino#47]
Just wondering if anyone else has worked on this yet. I'm wanting to use instrumentino to control an experiment that uses DS18B20 digital thermometers, but they require the DallasTemperature and OneWire libraries to function properly.

**Comment by yoelk** [comment]
Is it I2C? I think there is support for the wire module. look in the code.
Sorry for not supporting more, haven't worked on it for a long time..

**Comment by miqi95** [comment]
1-Wire is actually a different protocol, similar in concept to i2c but uses only 1 wire connection with a pullup. Comes on lots of devices from Maxim Integrated. I can try to work in library support myself, but I'm fairly novice with this code and given the popularity of the DS18B20 I'm sure someone has used it before.

Did interest or use of this project dry up? I wondered if the Kivy plans or another app had taken over the space.

**Comment by yoelk** [comment]
Yeah, I used to use Instrumentino a lot when I did my PhD, but as I already
graduated, and nobody else picked it up, it unfortunately doesn't get
maintained. Kivy was almost working but for the same reason, it stopped.
